src/server/svr_func.c

```diff
@@ -7178,20 +7193,25 @@ set_sched_throughput_mode(attribute *pattr, void *pobj, int actmode)
 int
 action_opt_bf_fuzzy(attribute *pattr, void *pobj, int actmode)
 {
-	char *str = pattr->at_val.at_str;
-
-	if (str == NULL)
-		return PBSE_BADATVAL;
+	if ( pobj == dflt_scheduler) {
```

Comment:
move the condition a line below because "str" will be needed in case of other schedulers too.